fineract-provider/src/main/java/org/apache/fineract/accounting/provisioning/service/ProvisioningEntriesReadPlatformServiceImpl.java

```diff
@@ -349,5 +349,37 @@ public String schema() {
         return this.loanProductProvisioningEntryDataPaginationHelper.fetchPage(this.jdbcTemplate, sqlCountRows, sqlBuilder.toString(),
                 whereClauseItemsitems, mapper);
     }
+    @Override
+	public Page<ProvisioningEntryData> retrieveAllProvisioningEntriesByDate(Integer offset, Integer limit,
+			String filterDate) {
+		Date filterdate = null;
+		if (filterDate != null) {
+			try {
+				SimpleDateFormat format = new SimpleDateFormat("dd MMMM yyyy", Locale.ENGLISH);
```

Comment:
Using Locale.ENGLISH? What if the user sent a different language? I think there is a locale param been sent on almost all request in Fineract..Maybe use that?